package-lock.json

```diff
@@ -31,6 +31,271 @@
         }
       }
     },
+    "@angular-devkit/build-angular": {
+      "version": "0.1100.5",
```

Comment:
are you sure you need to edit that file? We had a build crashes related to that file recently

Reply:
I'm not sure, but I didn't want to run the project without it